index.js

```diff
@@ -336,7 +336,7 @@
 						colSum
 					);
 
-					$targetCol.children[0]?.appendChild($colSum);
+					$targetCol.children[0]?.prepend($colSum);
```

Comment:
I think this always has been a bug; as the stuff that should float must appear first in the HTML.